modules/tinymce/src/core/test/ts/browser/EditorManagerCommandsTest.ts

```diff
@@ -36,6 +36,7 @@ describe('browser.tinymce.core.EditorManagerCommandsTest', () => {
   ], (test) => {
     it(`mceToggleEditor (${test.label})`, (done) => {
       viewBlock.update('<textarea id="ed_1" class="tinymce"></textarea>');
+      // eslint-disable-next-line @typescript-eslint/no-floating-promises
```

Comment:
Many of these are incorrect - what we should instead do is `await` the line below and make the test async. Which we did a month ago on the TINY-11177 branch 🫣 

At least now I can just search for the disable comment to find the rest of the places to fix...

Reply:
wait, my apologies, I was looking at the conflict before I properly understood the code.

These are callback tests, so the change to `await EditorManager.init()` is now duplicated by the call to `done()` in the init instance callback.

The question now is which test style will throw the correct error if the asserts fail 🤔

Reply:
yeaaaaah this never failed correctly because `init_instance_callback` isn't called during the `EditorManager.init()` promise. Doesn't matter if we use the new one or the old one.

It at least does fail the build with the new bedrock, because it finally catches the unhandled promise rejection correctly. The error is nonsense but at least there is _an_ error.

To fail correctly, we'd need some sort of helper that
* made a promise
* called `EditorManager.init()`
* resolved inside `init_instance_callback`
(I think I've seen code like that somewhere recently, so it shouldn't be a difficult pattern to extract

Then we just `await initInstance()` and run the assertions afterwards.

cc @shanmen-tiny

Reply:
Oops, I'll fix this in the TINY-11177 branch. 

I think this should be the pattern: https://github.com/tinymce/tinymce/blob/3bfbda75dd7ceb63e166f04e2564b5f5b3e32324/modules/tinymce/src/core/test/ts/browser/EditorAutoFocusTest.ts#L40

Resolving within `init_instance_callback` instead of on focus.

Reply:
Sounds like something that should go in mcagar 😂